Mono.Cecil.Cil/PortablePdb.cs

```diff
@@ -171,7 +171,7 @@ public ISymbolReader GetSymbolReader (ModuleDefinition module, string fileName)
 				throw new InvalidOperationException ();
 
 			return new EmbeddedPortablePdbReader (
-				(PortablePdbReader) new PortablePdbReaderProvider ().GetSymbolReader (module, GetPortablePdbStream (entry)));
+				(PortablePdbReader)new PortablePdbReaderProvider ().GetSymbolReader (module, GetPortablePdbStream (entry)));
```

Comment:
Please undo any line damage or formating changes, they make upstream merges pain

Reply:
Apparently I hit something in VS which reformatted the entire file. I'm so glad we have lint in dotnet/linker :wink:.
Anyway - hopefully I was able to revert all of these now.